.github/actions/argus-builder/docker-build/action.yml

```
@@ -111,8 +111,9 @@ runs:
         secret-files: ${{ inputs.secret_files }}
         load: true
     - name: Scan for vulnerabilities
-      uses: chanzuckerberg/github-actions/.github/actions/container-scanning@193e90c0872c379105b55b1d02cf55e9992f6f16
+      uses: chanzuckerberg/github-actions/.github/actions/container-scanning@a8769e107f2f0d6377933835baaa370a60295724
```

```suggestion
      uses: chanzuckerberg/github-actions/.github/actions/container-scanning@v5
```

Soon we'll be able to pin just that action's version